strings/ru.py

```diff
@@ -114,7 +114,8 @@
 # Conversation: confirm promotion to admin
 conversation_confirm_admin_promotion = "Вы уверены, что хотите повысить этого пользователя до 💼 Менеджера?\n" \
                                        "Это действие невозможно отменить!"
-
+# Conversation: language select menu header
+conversation_language_select = "Выберите язык:"
```

Comment:
Can you please add a newline here?

```suggestion
conversation_language_select = "Выберите язык:"

```